### ISSUE TITLE
Add Log options to settings.

### ISSUE THREAD

**Issue by aha999** [fr3ts0n/AndrOBD#154]
**Is your feature request related to a problem? Please describe.**
Currently we have to manually search for a log file if we want to include it in a report.

**Describe the solution you'd like**
If the developer settings contained this menu, it would be more convenient in handling log features imo:
Logs:
- Export log file
- Open Log file directory, which would open directory in file manager
- Share log file
- View log file ( if possible )

**Describe alternatives you've considered**
/

**Additional context**
/

**Comment by fr3ts0n** [comment]
I agree to a function to view the log directory so it does not to be searched.
Viewing the current log file might already produce system problems, since it is viewed while entries get appended.

Functions like share- or export log files I would NOT integrate into the app, since I want to have it clean of any potential spy functionality!
- Logs will only be generated locally
- If someone wants to have the logs analyzed, they will have to be sent / attached manually!
- No call home on anything!
